.github/Person.java

```diff
@@ -0,0 +1,30 @@
+public class Person {
+	String name;
+	String address;
+
+	Person(String name, String address) {
+		this.name = name;
+		this.address = address;
+	}
+
+	String getName(String name) {
+		return name;
+	}
+
+	String getAddress(String address) {
+		return address;
+	}
+
+	void setAddress(String address) {
+		address = "Cirebon";
+	}
+
+	public String toString() {
+		return String.format(getName(name) + "(" + getAddress(address) + ")");
```

Comment:
bisa langsung aja, gaperlu string format lagi
untuk manggil getName / getAddress juga ga perlu sepertinya ya soalnya bisa akses name nya langsung
```suggestion
		return  this.name + "(" + this.address + ")";
```

Reply:
ok maskasih mas feedbacknya